testsuite/shared/src/main/java/org/jboss/as/test/deployment/DeploymentInfoUtils.java

```diff
@@ -0,0 +1,797 @@
+        check(new InternalParameters(result).setName(name).setExpectedState(expected).setCtx(ctx));
+    }
+
+    /**
+     * Checking for non existence of application deployment
+     *
+     * @param result Result of called list/info command requested for checking
+     * @param name   Represent name of application deployment for testing
+     * @throws CommandFormatException Throw in case of assert failure
+     * @throws IOException            Throw in case of problem with execute management command
+     */
+    public static void checkMissing(DeploymentInfoResult result, String name) throws CommandFormatException, IOException {
+        check(new InternalParameters(result).setName(name).setSearchType(InternalParameters.SearchType.MISSING));
+    }
+
+    /**
+     * Checking for non existence of application deployment.
+     *
+     * @param result Result of called list/info command requested for checking.
+     * @param name   Represent name of application deployment for testing.
+     * @param ctx    Represent CommandContext to cli connection and handle management commands
+     * @throws CommandFormatException Throw in case of assert failure
+     * @throws IOException            Throw in case of problem with execute management command
+     */
+    public static void checkMissing(DeploymentInfoResult result, String name, CommandContext ctx) throws CommandFormatException, IOException {
+        check(new InternalParameters(result).setName(name).setSearchType(InternalParameters.SearchType.MISSING).setCtx(ctx));
+    }
+
+    /**
+     * Checking for empty installed deployments.
+     *
+     * @param result Result of called list/info command requested for checking.
+     */
+    public static void checkVoid(DeploymentInfoResult result) {
+        if (result == null) {
+            throw new IllegalArgumentException("Cant check null to void!");
+        }
+        assertThat("Command output contains some deployments! Checking a void FAILED", result.isOutputEmpty(), is(true));
+    }
+
+    /**
+     * In case you need know state of application deployment
+     *
+     * @param result Result of called list/info command requested for checking
+     * @param name   Name of application deployment
+     * @return State of application deployment, if not found return UNKNOWN state
+     * @throws CommandFormatException
+     */
+    public static DeploymentState getStateByOutputMemory(DeploymentInfoResult result, String name) throws CommandFormatException {
+        result.isOutputEmpty();
+
+        for (String row : result.getRows()) {
+            if (row.contains(name)) {
+                final DeploymentState[] statuses = DeploymentState.values();
+
+                for (DeploymentState state : statuses) {
+                    if (row.contains(state.getTitle())) {
+                        log.info("Application deployment are state '" + name + "'->'"
```

Comment:
Application deployment are state ==> Application deployment state

Reply:
Thank you. Done